boxoffice/views/item_collection.py

```diff
@@ -0,0 +1,66 @@
+from flask import render_template, jsonify
+from flask.ext.cors import cross_origin
+from coaster.views import load_models
+from boxoffice import app, ALLOWED_ORIGINS
+from boxoffice.models import Organization, ItemCollection, Price
+from utils import xhr_only
+
+
+def jsonify_item(item):
+    price = Price.current(item)
+    if price:
+        return {
+            'name': item.name,
+            'title': item.title,
+            'id': item.id,
+            'description': item.description,
+            'quantity_available': item.quantity_available,
+            'quantity_total': item.quantity_total,
+            'category_id': item.category_id,
+            'item_collection_id': item.item_collection_id,
+            'price': price.amount,
+            'price_category': price.title,
+            'price_valid_upto': price.valid_upto,
+            'discount_policies': [{'id': policy.id, 'title': policy.title, 'is_automatic': policy.is_automatic()}
+                                  for policy in item.discount_policies]
+        }
+
+
+def jsonify_category(category):
+    category_items = []
+    for item in category.items:
+        item_json = jsonify_item(item)
+        if item_json:
+            category_items.append(item_json)
+    if category_items:
+        return {
+            'id': category.id,
+            'title': category.title,
+            'name': category.name,
+            'item_collection_id': category.item_collection_id,
+            'items': category_items
+        }
+
+
+@app.route('/boxoffice.js')
+@cross_origin(origins=ALLOWED_ORIGINS)
+def boxofficejs():
+    return render_template('boxoffice.js',
+                           base_url=app.config.get('BASE_URL'),
+                           razorpay_key_id=app.config.get('RAZORPAY_KEY_ID'))
+
+
+@app.route('/<organization>/<item_collection>', methods=['GET'])
```

Comment:
Don't put `/<organization>` at the root level as this is not a user-facing app. We don't need pretty URLs here and there's too much scope for conflict at the top level.